app/main.py

```diff
@@ -45,9 +49,11 @@ def upload():
             flash('Please select a PNG image to Upload!', 'error')
             return redirect(url_for('index'))
 
+    # If the csv file is uploaded
     if '.' in filename and filename.rsplit('.', 1)[1] == 'csv':
         filename = secure_filename(filename)
-        file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+        if filename != "default.png.csv":
```

Comment:
This should never be the case, right?

Reply:
I mean, `filename != "default.png.csv"` is always the case, right?

Reply:
Yes. We will replace the name `default.png.csv` with a random long string so that it eliminates the possibility of collision.